README.md

```diff
@@ -27,9 +26,9 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [PSQL] <a name="about-project"></a>
+# 📖 [Vet Clinic] <a name="about-project"></a>
 
-**[PSQL]** is a Postgre SQL project with queries of doing CRUD operations using Postgre CLI.
+**[Vet Clinic]** is a Postgre SQL project with queries of doing CRUD operations using Postgre CLI.
 
 ## 🛠 Built With <a name="built-with"></a>
 
```

Comment:
- [ ] Kindly fix the broken link to your `issues` page in the readme file as currently it takes the user to a 404 page